src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,47 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH = 10;
+    private final K[] keyStorage;
+    private final V[] valueStorage;
+    private int count = 0;
```

Comment:
```suggestion
    private int count;
```
the default value of int is 0, so we can omit setting it